cdap-docs/build.sh

```diff
@@ -30,6 +30,34 @@ ARG_1="${1}"
 ARG_2="${2}"
 ARG_3="${3}"
 
+function usage() {
+  cd ${PROJECT_PATH}
+  PROJECT_PATH=`pwd`
+  echo "Build script for '${PROJECT_CAPS}' docs"
+  echo "Usage: ${SCRIPT} <option> [source]"
+  echo ""
+  echo "  Option (select one)"
+  echo "    docs-all  Clean build of everything: HTML and Javadocs, GitHub and Web versions, zipped"
+  echo ""
+  echo "    docs-github-only  Clean build of HTML, zipped, with GitHub code, skipping Javadocs"
+  echo "    docs-web-only     Clean build of HTML, zipped, with docs.cask.co code, skipping Javadocs"
+  echo "    docs              Dirty build of HTML with docs.cask.co code, skipping zipping and Javadocs"
+  echo ""
+  echo "    docs-github  Clean build of HTML and Javadocs, zipped, for placing on GitHub"
+  echo "    docs-web     Clean build of HTML and Javadocs, zipped, for placing on docs.cask.co webserver"
+  echo ""
+  echo "    clean     Clean up any previous build's target directories"
+  echo "    javadocs  Build Javadocs"
+  echo "    licenses  Clean build of License Dependency PDFs"
+  echo "    sdk       Build CDAP SDK"
+  echo "    version   Print the version information"
+  echo ""
+  echo "  with"
+  echo "    source    Path to ${PROJECT} source, if not '${PROJECT_PATH}'"
+  echo "              Path is relative to '${SCRIPT_PATH}/../..'"
+  echo ""
```

Comment:
I thought for sure I ran into a problem with using ``echo`` versus ``echo ""``, but...